Assets/Scripts/BulletBehaviour.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletBehaviour : MonoBehaviour
+{
+    public int bulletType;
```

Comment:
consider looking into using an `Enum` for the bullet type. Would probably show up as a dropdown menu in the unity inspector, which is a lot more useful than a magic number

Reply:
changed to enum. Will have to check for bugs